testing/run-scripts/serve-zork-metrics.sh

```diff
@@ -0,0 +1,4 @@
+docker build -t zork-metrics ../../monitor
+HOST_IP=`ip -o -4 addr list docker0 | awk '{print $4}' | cut -d/ -f1`
+docker run -d -p 8000:8000 --add-host zork:$HOST_IP zork-metrics
```

Comment:
Did you try `--link`? It was designed for cases just like this - and you won't need the `HOST_IP` hack. Try:
```bash
docker run -d -p 8000:8000 --link=uproxy-zork:zork zork-metrics
```

More info here:
https://docs.docker.com/v1.8/articles/networking/